### PR TITLE
Form with validation

### PR DESCRIPTION
**Pull request by p-nedelchev**
Task: Create HTML form with following fields: First name, Last name, Egn, Age, Password, Confirm password. Validate the form using JS.

### CODE REVIEW

**Comment by p-nedelchev** [comment]
Needs review

File: taskthree/script.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by Peter Nedelchev <peter.krasimirov@gmail.com>.
+ */
+
+function validator(form) {
+    validate(/^[a-zA-z]{1,15}/, form.firstname.value, 'firstname_alert', 'Bad name');
+    validate(/^[a-zA-z]{1,15}/, form.lastname.value, 'lastname_alert', 'Bad name');
+    validate(/\d{10}/, form.egn.value, 'egn_alert', 'Egn is exact 10 digits');
+    if(form.age.value < 18 || 118 > form.age.value) {
```
Comment (tsony-tsonev) [inline taskthree/script.js]: you can do this with regex also
```javascript
^(1[89])$|^([2-9][0-9])$|^(10[0-9])$|^(11[0-8])$
```

File: taskthree/script.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/**
+ * Created by Peter Nedelchev <peter.krasimirov@gmail.com>.
+ */
+
+function validator(form) {
+    validate(/^[a-zA-z]{1,15}/, form.firstname.value, 'firstname_alert', 'Bad name');
```
Comment (tsony-tsonev) [inline taskthree/script.js]: use camelCase everywhere

File: taskthree/script.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    validate(/^[a-zA-z]{1,15}/, form.lastname.value, 'lastname_alert', 'Bad name');
+    validate(/\d{10}/, form.egn.value, 'egn_alert', 'Egn is exact 10 digits');
+    if(form.age.value < 18 || 118 > form.age.value) {
+        document.getElementById("age_alert").innerHTML = "Incorrect age";
+        return;
+    }
+    validate(/^[a-zA-z0-9]{1,100}/, form.address.value, 'address_alert', 'Address ');
+    validate(/^([a-zA-z0-9]{6,18})$/, form.password.value, 'password_alert', 'Only letters and digits');
+    if (password != confirmedPass) {
+        document.getElementById("passconf_alert").innerHTML = "Password doesn't match";
```
Comment (tsony-tsonev) [inline taskthree/script.js]: you should set empty string if they match, cause you can change your passwords to match but mess-up something else and this message will stay unchanged

**Comment by tsony-tsonev** [comment]
LGTM after fixing the comments

**Pull request merged**